### PR TITLE
Implement Adding Members To A Channel

### PR DESCRIPTION
**Pull request by ugoostanleyibe**
I have implemented the adding of members to a channel as can be seen in the gif below:

demo3

### CODE REVIEW

File: lib/ui/view/channel/add_people/channel_add_people_viewmodel.dart [hunk 1]
```diff
@@ -34,24 +36,32 @@ class ChannelAddPeopleViewModel extends BaseViewModel {
...
+    matchingUsers =
+        users = (await organizationApi.fetchMembersInOrganization(orgId!))
+          ..sort((user1, user2) =>
+              user1.userName!.trim().compareTo(user2.userName!.trim()));
     setBusy(false);
-    _navigationService.popRepeated(1);
   }
 
-  Future<void> addMemberToChannel(String channelId, String userId) async {
+  /* Future<void> addMemberToChannel(String channelId, String userId) async {
```
Comment (pauleke65) [inline lib/ui/view/channel/add_people/channel_add_people_viewmodel.dart:64]: Delete unused code

**Review by pauleke65 — CHANGES_REQUESTED** [review]
Remove unused code.
and replace the video with one showing the number of members on the channel page updating after you've added new members

File: lib/ui/view/channel/add_people/channel_add_people_viewmodel.dart (hunk 1 above)
Comment (ugoostanleyibe) [reply lib/ui/view/channel/add_people/channel_add_people_viewmodel.dart]: I have done that!

**Comment by jeremiahseun** [comment]
Resolve conflicts please

**Comment by ugoostanleyibe** [comment]
> Resolve conflicts please

I'm on it right now!

**Comment by ugoostanleyibe** [comment]
> Resolve conflicts please

I have resolved them now!

**Comment by ugoostanleyibe** [comment]
> Remove unused code. and replace the video with one showing the number of members on the channel page updating after you've added new members

I've added a new gif as well 🙃

**Review by pauleke65 — APPROVED** [review]
Awesome Work

**Review by YenumJay — APPROVED** [review]
Great Job

**Comment by ugoostanleyibe** [comment]
> Awesome Work

Thanks!!! 😊

**Comment by ugoostanleyibe** [comment]
> Great Job

Thank you!! 😊

**Pull request merged**